core/runtime/src/main/scala/com/fortysevendeg/exercises/MethodEval.scala

```diff
@@ -65,8 +65,8 @@ class MethodEval() {
   val timeout = 20.seconds
   private val evaluator = new Evaluator(timeout)
 
-  def eval[T](qualifiedMethod: String, rawArgs: List[String], imports: List[String] = Nil): EvaluationResult[_] = {
-    val pre = imports.mkString(System.lineSeparator)
+  def eval[T](pkg: String, qualifiedMethod: String, rawArgs: List[String], imports: List[String] = Nil): EvaluationResult[_] = {
+    val pre = (s"import $pkg._" :: imports).mkString(System.lineSeparator)
```

Comment:
there is a tiny tiny deference of semantic:

`import a._` has access to less scope than `package a`.

```
AccessModifier    ::=  (‘private’ | ‘protected’) [AccessQualifier]
AccessQualifier   ::=  ‘[’ (id | ‘this’) ‘]’
```
> -http://www.scala-lang.org/files/archive/spec/2.11/13-syntax-summary.html

see also http://www.jesperdj.com/2016/01/08/scala-access-modifiers-and-qualifiers-in-detail/

I'm not sure what would be the most correct semantic for us.